documentation.md

```diff
@@ -0,0 +1,344 @@
+
+> Solution object (send): {
+> "id": integer,
+> "date": string (MM_DD_YY),
+> "time": string (HH:MM AM/PM)
+> }
+
+- PUT to [https://classroom-angel-development.herokuapp.com/api/solutions/id](https://classroom-angel-development.herokuapp.com/api/solutions/id) - update a specific solution
+
+| URL Params | Body                   | Returns                  |
+| ---------- | ---------------------- | ------------------------ |
+| ID         | Solution object (send) | Solution object (return) |
+
+- DELETE to [https://classroom-angel-development.herokuapp.com/api/solutions/id](https://classroom-angel-development.herokuapp.com/api/solutions/id) - delete a specific solution
+
+| URL Params | Body | Returns                  |
+| ---------- | ---- | ------------------------ |
+| ID         | N/A  | Solution object (return) |
+
+## Tags
+
+- GET to [https://classroom-angel-development.herokuapp.com/api/tags/](https://classroom-angel-development.herokuapp.com/api/tags/) - retrieve all tags
+
+| URL Params | Body | Returns                  |
+| ---------- | ---- | ------------------------ |
+| N/A        | N/A  | All tag objects (return) |
+
+> Tag object (return): {
+> "id": integer,
+> "name": string
+> }
+
+- POST to [https://classroom-angel-development.herokuapp.com/api/tags/](https://classroom-angel-development.herokuapp.com/api/tags/) - create a tag
+
+| URL Params | Body              | Returns             |
+| ---------- | ----------------- | ------------------- |
+| N/A        | Tag object (send) | Tag object (return) |
+
+> Tag object (send): {
+> "id": integer,
+> "name": string
+> }
+
+- PUT to [https://classroom-angel-development.herokuapp.com/api/tags/id](https://classroom-angel-development.herokuapp.com/api/tags/id) - update a specific tag
+
+| URL Params | Body              | Returns             |
+| ---------- | ----------------- | ------------------- |
+| ID         | Tag object (send) | Tag object (return) |
+
+- DELETE to [https://classroom-angel-development.herokuapp.com/api/tags/id](https://classroom-angel-development.herokuapp.com/api/tags/id) - delete a specific tag
+
+| URL Params | Body | Returns             |
+| ---------- | ---- | ------------------- |
+| ID         | N/A  | Tag object (return) |
+
+## Users
+
+- GET to [https://classroom-angel-development.herokuapp.com/api/users/](https://classroom-angel-development.herokuapp.com/api/users/) - retrieve all users
+
+| URL Params | Body | Returns                   |
+| ---------- | ---- | ------------------------- |
+| N/A        | N/A  | All user objects (return) |
+
+> User object (return): {
+> "id": integer,
+> "username": string,
+> "first_name": string,
+> "last_name": string,
+> "password": string,
+> "role": string,
+> "organization_id": integer
+> }
+
+- POST to [https://classroom-angel-development.herokuapp.com/api/register/](https://classroom-angel-development.herokuapp.com/api/register/) - register a user
```

Comment:
I believe this should be `api/users/register`

Reply:
Done